roles/swift-common/tasks/main.yml

```diff
@@ -35,7 +35,7 @@
     depth=50
     update={{ openstack.git_update }}
   notify:
-    - pip install swifttool
+    - pip install swifttool - {{ openstack_install_method }}
```

Comment:
If you collapse the two tasks above into one, this can go back to the generic handler name.